File: backend/internal/model/common/investment.go
package model

type Investment struct {
	ID         *int64 `json:"id,omitempty"`
	PitchID    *int64 `json:"pitch_id,omitempty"`
	InvestorID string `json:"investor_id"`
	TierID     *int64 `json:"tier_id,omitempty"`
	Amount     int64  `json:"amount"`
	Refunded   bool   `json:"refunded"`
	CreatedAt  string `json:"created_at,omitempty"`
}

File: frontend/lib/types/investment.ts
export type Investment = {
	investment_id: number;
	pitch_id: number;
	investor_id: number;
	amount: number;
	refunded: boolean;
	created_at: Date;
};

export type InvestmentFromApi = {
	id: number;
	pitch_id: number;
	amount: number;
	created_at: string;
	investor_id: string;
	tier_id: number;
};
